### DISCUSSION TITLE
Lost Email To GitHub Account

### DISCUSSION THREAD

**Discussion by mochavessel** [community/community#69243] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, a few days ago I reset my PC without remembering to backup, thinking all my applications would be saved. However, they were not. When I tried to sign into GitHub, my password was correct but it asked me to get my 2FA from my backup email, which I have forgotten, due it to being a string of numbers. Is there anything I can do to recover my main GitHub account?

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @mochavessel ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by mochavessel** [reply]
I've already tried there, and have not found an answer..
